proc_IR.py

```diff
@@ -77,16 +56,17 @@
     M0,Mi,R1,vd = sympy.symbols("M_0 M_inf T_1 indirect", real=True)
     s.functional_form = Mi + (M0-Mi)*sympy.exp(-vd*R1)
     logger.info(strm("Functional form", s.functional_form))
+    s.fit_coeff = r_[-1,1,1]
     # JF notes that we want to be able to set the guess using a dictionary
     # here (which is what I think setting fit_coeff was doing), then plot
     # the guess to make sure that's what we're doing -- like so
     fl.next('t1 test')
-    s.set_guess({M0:-1, Mi:1, R1:1}) # this is the only line that will not
+    #s.set_guess({M0:-1, Mi:1, R1:1}) # this is the only line that will not
     # work, currently -- we will need a pull request on pyspecdata as well
     # to make it work
     fl.plot(s, 'o', label=s.name())
-    s.settoguess()
-    fl.plot(s, '-', label='initial guess')
+    #s.settoguess()
+    #fl.plot(s, '-', label='initial guess')
```

Comment:
You just commented this out.  The point was that to get this to work, we need changes on pyspecdata to make it work.  Please just do this.

Reply:
maybe go back and re-read the notes in the [original conversation](https://github.com/jmfrancklab/proc_scripts/pull/14#discussion_r441806730)

Reply:
maybe go back and re-read the notes in the [original conversation](https://github.com/jmfrancklab/proc_scripts/pull/14#discussion_r441806730)